README.md

```diff
@@ -62,8 +62,29 @@ E.g. `feat: RTENU-12345 Awesome new feature`
 Check node version. You can use `nvm use` to automatically set the right version.
 Run ` npm i`
 
+To install a common dependency that both frontend and server can use, run command in the root repository:
+
+```
+npx lerna add *npm_package_name*
+npx lerna add *npm _package_name* --dev
+```
+
 ### Usage
 
+#### Frontend development
+
+```
+cd packages/frontend
+npm start
```

Comment:
Maybe we could condense this to be started from the root level to streamline it a bit. I.e. skip the cd and do something like `npm run fe` and `npm run be` or something (I'm not totally happy with my suggested script names but you get the gist, right?). Not necessary to do at this point, but something to keep in mind.